projects/ngx-charts-on-fhir/src/lib/analysis-cards/analysis-card/analysis-card.component.css

```diff
@@ -7,4 +7,9 @@
 }
 .mat-card-title {
   font-size: 20px;
+  max-width: 250px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  cursor: pointer;
```

Comment:
`cursor: pointer` should only be used for clickable elements. The card title does nothing when you click on it. This will confuse users.